index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="header">
+   <div class="head">
+    <a href="#" class="link"><div class="logo">My Logo</div></a>
+   </div>
+   <nav class="menubar">
+    <img src="Media/Union.svg" class="menu"></div>
+  </header>  
+  <section class="headline">
+  <div class="me">
+    <h1>I’m Sadaf.
+    <span>Glad to see you!</span></h1>
+  </div>
+  <div class="dec"></div>
+    <p>I’m Sadaf.
+    Glad to see you!
+    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+     </p></div>
+     <h2 class="social">LET’S CONNECT</h2> 
+     <div class="social_icon">
+       <a href="https://twitter.com/SadafDaneshga" class="twitter"><img src="Media/twitter.svg"></a>
+       <a href="https://www.linkedin.com/in/sadaf-barekzai-00480a242" class="linkindin" ><img src="Media/Linkedin icon.svg" alt="Linkdin"></a>
+       <a href="sadaf.barekzai54321@gmail.com" class="mail"><img src="Media/mediu.svg" alt="Mail"></a>
+       <a href="https://github.com/sadaf-Daneshgar" class="github"><img src="Media/github.svg"  alt="Github"></a>
+      <a href="#" class="angellist"> <img src="Media/angellist.svg" alt="Anlgelist"></a>
+     </span>
```

Comment:
- [ ] Please consider using the `ul` tag as a container and put all the images in individual `li` tags, it will make your code more professional and semantically correct. It will also help **visually-impaired** people to navigate through your website with a good *User Experience*.
- [ ] Please make sure that always close your tag with the right `tagName` otherwise as your code scale or as you will add more and more features to it, later on, it will difficult for you to find the error just because of a *small* mistake.